lib/imagec/docker.go

```diff
@@ -364,12 +364,7 @@ func FetchImageManifest(op trace.Operation, options Options, schemaVersion int,
 		return nil, "", err
 	}
 
-	// Cleanup function for the error case
-	defer func() {
-		if err != nil {
-			os.Remove(manifestFileName)
-		}
-	}()
+	defer os.Remove(manifestFileName)
```

Comment:
Would the `manifestFile` ever be used after the `FetchImageManifest` function?

Reply:
No, it will not.